spec/rdiff-backup-test/create_server_spec.rb

```diff
@@ -126,6 +148,18 @@
         )
       end
       it do
+        expect(chef_run).to create_file(
+          '/home/rdiff-backup-server/exclude/192.168.60.12/_test2'
+        ).with(
+          owner: 'rdiff-backup-server',
+          group: 'rdiff-backup-server' || 'rdiff-backup-server',
```

Comment:
I'm not sure, I just copied and pasted it from another part. I can likely remove it.